applications/nrf5340_audio/src/bluetooth/bt_stream/bt_le_audio_tx/bt_le_audio_tx.c

```diff
@@ -217,28 +225,30 @@ int bt_le_audio_tx_send(struct bt_bap_stream **bap_streams, uint8_t *audio_mappi
 	bool ts_common_acquired = false;
 	uint32_t common_interval = 0;
 
-	for (int i = 0; i < streams_to_tx; i++) {
-		if (tx_info_arr[i].iso_tx.seq_num == 0) {
+	for (int i = 0; i < num_tx; i++) {
+		if (tx_info_arr[tx[i].idx.lvl1_idx][tx[i].idx.lvl2_idx][tx[i].idx.lvl3_idx]
```

Comment:
Can we make this more readable?

Reply:
We could do
```suggestion
                int seq_num = tx_info_arr[tx[i].idx.lvl1_idx][tx[i].idx.lvl2_idx][tx[i].idx.lvl3_idx].iso.tx.seq_num;
		if (seq_num == 0)
```
Not sure if that helps though

Reply:
Remove idx from level in stream_index struct